python/pyspark/ml/tests.py

```diff
@@ -640,6 +640,23 @@ def test_count_vectorizer_with_binary(self):
             feature, expected = r
             self.assertEqual(feature, expected)
 
+    def test_count_vectorizer_from_vocab(self):
+        model = CountVectorizerModel.from_vocabulary(["a", "b", "c"], inputCol="words",
```

Comment:
Good first test, I'd love to also see it with empty vocab, and also one that uses the default values.

Reply:
Yeah, good idea, I'll add those

Reply:
done